internal/api/tests/private/version_test.go

```diff
@@ -0,0 +1,40 @@
+package private
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+
+	"playbook-dispatcher/internal/common/config"
+	"playbook-dispatcher/internal/common/utils/test"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Version", func() {
+	Describe("get internal version", func() {
+		It("should return the git revision the API was built from", func() {
```

Comment:
Better than to rely on the defaults is to set the test value explicitly in https://github.com/RedHatInsights/playbook-dispatcher/blob/fd5ea274899a1cd0d6eb7f8c7bac5e3707904151/internal/api/tests/private/private_suite_test.go#L46 and then test for the given value.